addon/components/challenges/issue.js

```diff
@@ -3,7 +3,25 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   layout,
-  isPending: false,
-  isFailure: false,
-  isSucces: false
+  getTitle: Ember.computed('model', function() {
+    let title = this.get('model.title');
+    if(title.length > 50){
+      return title.substr(0,42) + '...';
+    }
+    else
+    {
+      return title;
+    }
+  }),
+
+  idSize: Ember.computed('model', function() {
```

Comment:
This is not the way to go I think. The size between cards will then differ which is not very neat. Perhaps figure out the longest repository name and take that as a reference for the maximum font size. And perhaps put a ellipsis on it when it is even long than that so it can be seen with a hover.